en/flight_modes_rover/manual.md

```diff
@@ -0,0 +1,88 @@
+
+- [**Ackermann Rover**](../frames_rover/ackermann_rover.md): Moving the left-stick up/down controls the _forward speed_ and moving the right-stick left/right controls the _steering angle_ of the vehicle.
+
+::: info
+The rover does not attempt to maintain a specific orientation or compensate for external factors like slopes or uneven terrain!
+The user is responsible for making the necessary adjustments to the stick inputs to keep the rover on the desired course.
+:::
+
+## Acro Mode
+
+<Badge type="tip" text="Differential" />
+
+::: warning
+Acro mode is not supported for Ackermann rovers (only differential rovers).
+:::
+
+::: info
+This mode requires a yaw rate estimate.
+:::
+
+Acro Mode is similar to [Manual mode](#manual-mode), but with closed-loop yaw rate control.
+In this mode, the left-stick input remains open-loop for forward speed control, while the right-stick input commands a desired yaw rate setpoint, which is then maintained by the rover's closed-loop control system.
+
+See the tuning for [Acro mode](../frames_rover/differential_rover.md#acro-mode) section to go through the necessary setup to use Acro mode for differential rovers.
+
+## Stabilized Mode
+
+<Badge type="tip" text="Differential" />
+
+::: warning
+Stabilized mode is not supported for Ackermann rovers (only differential rovers).
+:::
+
+::: info
+This mode requires a yaw rate and yaw estimate.
+:::
+
+As with [Acro mode](#acro-mode), the left-stick input remains open-loop for forward speed control, while the right stick input commands a desired yaw rate setpoint, which is then maintained by the rover's closed-loop control system.
+
+In addition, the rover will do closed loop yaw (or heading) control when the right stick is centered, or in other words if the yaw rate setpoint is zero.
+This means that the rover will maintain the current heading if the user only gives throttle input, but no yaw rate input.
+Compared to [Acro mode](#acro-mode), this mode is much better at driving in a straight line as it can more effectively reject disturbances.
+
+See the tuning for [Stabilized mode](../frames_rover/differential_rover.md#stabilized-mode) section to go through the necessary setup to use Acro mode for differential rovers.
+
+## Position Mode
+
+<Badge type="tip" text="Differential" />
+
+::: warning
+Stabilized mode is not supported for Ackermann rovers (only differential rovers).
+:::
+
+::: info
+This mode requires a yaw rate, yaw, speed and position estimate.
+:::
+
+This is the most advanced manual mode. It uses _closed loop yaw rate control_, _closed loop speed control_ and a special logic for driving a straight line when there is no yaw rate input.
+
+In this mode, moving the left-stick up/down commands a desired speed setpoint, and moving the right-stick left and right commands a desired yaw rate setpoint which are both close-loop controlled by the autopilot.
```

Comment:
1. What does setting a speed setpoint do for you in terms of changing the driving? Presumably if cornering the speed will adjust for the yaw setpoint?
2. Did you consider using an accel. setpoint? Matthias changed the MC position mode to this to make it "more like driving" - this seems less like driving.